### PR TITLE
Re-add docs fixes that were lost in 4.2->4.3 cut

### PR DESCRIPTION
**Pull request by awly**
Specifically:
- https://github.com/gravitational/teleport/pull/3562
- https://github.com/gravitational/teleport/pull/3685

### CODE REVIEW

File: docs/4.3/architecture/teleport_auth.md [hunk 1]
```diff
@@ -215,41 +217,6 @@ storage.
...
-   connection to the destination node.
-
-However, there are advantages of proxy-based session recording too. When
-sessions are recorded at the nodes, a root user can add iptables rules to
-prevent sessions logs from reaching the Auth Server. With sessions recorded at
-the proxy, users with root privileges on nodes have no way of disabling the
-audit.
-
-See the [admin guide](../admin-guide.md#recorded-sessions) to learn how to turn on the
-recording proxy mode.
```
Comment (benarent) [inline docs/4.3/architecture/teleport_auth.md:251]: Looks good, I removed the TOCs which are now in the sidebar. Any reason for this being removed?

Comment (awly) [reply docs/4.3/architecture/teleport_auth.md:251]: Thanks for cleaning up.
This is a duplicate of what already exists in https://gravitational.com/teleport/docs/architecture/teleport_proxy/#recording-proxy-mode (also removed in https://github.com/gravitational/teleport/pull/3562)

Comment (benarent) [reply docs/4.3/architecture/teleport_auth.md:251]: ty!

**Review by benarent — APPROVED** [review]

**Comment by webvictim** [comment]
Nice! 🚀

**Pull request merged**